src/sass/modules/_m-button.scss

```diff
@@ -174,4 +174,8 @@ button.short {
   margin: 0;
   text-indent: -9999rem;
   width: 1rem;
-}
\ No newline at end of file
+}
+
+.no-text-transform {
```

Comment:
We could put this in `src/sass/base/_b-utils.scss`. There's a similar `no-background-image` class in there.